webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/ResourceFileService.scala

```diff
@@ -115,4 +116,17 @@ class GitResourceFileService(gitReposProvider: GitRepositoryProvider) extends Re
       filesNotCommitted ::: untouched
     }
   }
+
+  override def cloneResourcesFromTechnique(draftId: String, techniqueId: String, techniqueVersion: String, category : String): IOResult[Unit] = {
+
+    IOResult.attempt(s"An error occurred while copying resources of technique ${techniqueId}/${techniqueVersion} to a new draft technique ${draftId}") {
+      val resourceDir = gitReposProvider.rootDirectory / s"techniques/${category}/${techniqueId}/${techniqueVersion}/resources"
+      val workspace = gitReposProvider.rootDirectory / s"workspace/${draftId}/${techniqueVersion}"
```

Comment:
please avoid strings for global vars. Ideally they should be services for easier testing, or at lest something provided centrally in an object.